src/main/java/com/theironyard/entities/Quote.java

```diff
@@ -0,0 +1,88 @@
+package com.theironyard.entities;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.theironyard.utilities.LocalDateTimeConverter;
+
+import javax.persistence.*;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.util.Collection;
+
+/**
+ * Created by vasantia on 8/1/16.
+ */
+
+@Entity
+@Table(name = "quotes")
+@JsonIgnoreProperties({"hibernateLazyInitializer", "handler"})
+public class Quote {
+
+    @Id
+    @GeneratedValue
+    private int id;
+
+    @Column
```

Comment:
`@Column` isn't doing anything in this code since it doesn't set any of the values like nullable